src/core/math/mat4/mirror.js

```diff
@@ -7,7 +7,7 @@ const fromValues = require('./fromValues')
  * create an affine matrix for mirroring into an arbitrary plane:
```

Comment:
mirror is complicated, our old & new implementations differ quite a bit

Reply:
I’ve been testing for consistent results as well. If we need to revisit mirror then now is the time. The math functions are the heart and soul of the library.

Do you have an example? I’m sure the API is broken, so we can skip that level.

Reply:
not off the top of my head but it is essential that we keep both implementations in the file for now, until they can be unified/ merged down the line